### PR TITLE
style: dropdown and input have same style when disabled

### LINKED ISSUE #492: Dropdown and Input have different styles when disabled
These two should look more similar:

image
image

IMO, the former (`Input`) component looks better when disabled so we can just tweak the styles of `Dropdown` component.

### PR DESCRIPTION
**Pull request by skarunakar**
Fixes #492.

### CODE REVIEW

File: src/components/Dropdown/Dropdown.js [hunk 1]
```diff
@@ -79,6 +79,11 @@ const SelectedItem = styled(Item)`
   div:last-child {
     margin-left: auto;
   }
+    ${props =>
+    props.disabled &&
+    `
+    background-color: ${props.theme.colors.gray50};
+  `};
```
Comment (skarunakar) [inline src/components/Dropdown/Dropdown.js]: dropdown-disabled

Dropdown disabled with above changes

File: src/components/Dropdown/Dropdown.js [hunk 2]
```diff
@@ -79,6 +79,8 @@ const SelectedItem = styled(Item)`
   div:last-child {
     margin-left: auto;
   }
+  ${props =>
+    props.disabled && `background-color: ${props.theme.colors.gray50};`};
 `;
```
Comment (skarunakar) [inline src/components/Dropdown/Dropdown.js]: dropdown-disabled
Dropdown disabled with this changes

Comment (skarunakar) [reply src/components/Dropdown/Dropdown.js]: input-disabled

Input when disabled

File: src/components/Dropdown/Dropdown.js [hunk 3]
```diff
@@ -20,8 +20,8 @@ const Item = styled.div`
     props.disabled &&
     `
     cursor: not-allowed;
-    color: ${props.theme.colors.gray200};
-    background-color: ${props.theme.colors.white};
+    color: ${props.theme.colors.gray600};
```
Comment (skarunakar) [inline src/components/Dropdown/Dropdown.js]: gray600 is color of placeholder text in input component when disabled. Not sure if it should be rgb(0, 0, 0) or something else

Comment (fbarl) [reply src/components/Dropdown/Dropdown.js]: `gray600` seems good!

File: src/components/Dropdown/Dropdown.js (hunk 2 above)
Comment (fbarl) [reply src/components/Dropdown/Dropdown.js]: Perfect, looks much better IMO :)

**Comment by bia** [comment]
Also looks like a great improvement to me, let’s merge. 👍 
Thank you @skarunakar !

**Pull request merged**